### PR TITLE
Fix build manger version using cnsi-manager/config/manager/kustomization.yaml change

### PR DESCRIPTION
**Pull request by qijianshuai**
Fix build manger version using cnsi-manager/config/manager/kustomization.yaml change
image
image

### CODE REVIEW

File: deployments/yaml/manager.yaml [hunk 1]
```diff
@@ -1323,7 +1323,7 @@ spec:
         - --leader-elect
         command:
         - /manager
-        image: projects.registry.vmware.com/cnsi/manager:0.3
+        image: projects.registry.vmware.com/cnsi/manager:0.3.2
```
Comment (bgubbala) [inline deployments/yaml/manager.yaml:1326]: Is this generated or hardcoded?

Comment (qijianshuai) [reply deployments/yaml/manager.yaml:1326]: hardcoded

Comment (bgubbala) [reply deployments/yaml/manager.yaml:1326]: I think this is supposed to be generated using `cnsi-manager/config/manager/kustomization.yaml` changes.
@4everming can you comment on this? Whether it has to be generated or not?

Comment (qijianshuai) [reply deployments/yaml/manager.yaml:1326]: @bgubbala I didn't understand what you meant before, and now I use cnsi-manager/config/manager/kustomization.yaml to change the generated manger version.

Comment (bgubbala) [reply deployments/yaml/manager.yaml:1326]: @qijianshuai  Your latest changes are Good. This is what I was expecting
Also, I have verified your latest changes on my local setup.

**Pull request merged**